diff --git a/dist/client/app/components/Content.js b/dist/client/app/components/Content.js
index 9ccd6e81204c558bd4f267b38ae806cf99d7bcaf..8e2306f010ebadf0d899a9002f5d0a62830b9d79 100644
--- a/dist/client/app/components/Content.js
+++ b/dist/client/app/components/Content.js
@@ -1,4 +1,4 @@
-import { defineComponent, h } from 'vue';
+import { defineComponent, h, resolveComponent } from 'vue';
 import { useData, useRoute } from 'vitepress';
 import { contentUpdatedCallbacks } from '../utils';
 const runCbs = () => contentUpdatedCallbacks.forEach((fn) => fn());
@@ -9,8 +9,11 @@ export const Content = defineComponent({
     },
     setup(props) {
         const route = useRoute();
+        const Metadata = resolveComponent('Metadata');
+        const isNote = route.path.startsWith(`/notes`);
         const { site } = useData();
         return () => h(props.as, site.value.contentProps ?? { style: { position: 'relative' } }, [
+            isNote ? h(Metadata) : null,
             route.component
                 ? h(route.component, {
                     onVnodeMounted: runCbs,